app/models/identity.rb

```diff
@@ -39,6 +39,8 @@ def self.build_from_omniauth(provider)
   end
 
   def email
-    auth_data_dump.info.email
+    return auth_data_dump.info.email if auth_data_dump&.info&.email
```

Comment:
Ah sorry, I misunderstood this earlier. 😅 I think this could be refactored like so:
```
auth_data_dump&.info&.email || "the message"
```

Reply:
thanks! will take a look, meanwhile format(..) added - the **%<provider>s** with the s at the end is required to succesfully parse a String, works good tested locally.